pkg/server/serverpb/admin.proto

```diff
@@ -383,6 +383,53 @@ message DrainResponse {
   // to a probe request (!shutdown && skip_drain) if another
   // drain request has been issued prior or asynchronously.
   bool is_draining = 2;
+  // drain_progress_indicator provides an estimation of how much state
```

Comment:
It looks like what you're really summing up here is the "number of things that would need to be done in order to drain, as of when the drain request took stock (before taking action)". This is good, I just don't see it reflected as clearly in the naming and comments. The gerund (is being shed) also makes it unclear whether it was already shed, or will be shed, or was partially shed? I think it will be easier to understand this if it's more explicit. Something like this:

>
drain_remaining_indicator measures, at the time of starting to process the corresponding drain request, how many actions to fully drain the node were deemed to be necessary. Some, but not all, of these actions may already have been carried out by the time this indicator is received by the client. The client should issue requests until this indicator first reaches zero, which indicates that the node is fully drained.

The various `reporter` comments could also be updated to reflect this verbiage; I find them ambiguous about what should be reported at the time of writing.